src/screens/TopicPageScreen/index.js

```diff
@@ -107,6 +106,12 @@ const TopicPageScreen = (props) => {
     extrapolate: 'clamp'
   });
 
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `opacityImage` variable is introduced to control the opacity of an image based on the scroll position. This seems like a good addition for improving user experience, but it's hard to evaluate without seeing how `opacityImage` is used in the rest of the code. If it's not used properly, it could lead to unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ah ya, this's from old code, we not using this anymore